Detectors/TPC/base/include/TPCBase/Point3D.h

```diff
@@ -1,9 +1,16 @@
+///
+/// @file   Point3D.h
+/// @author Jens Wiechula, Jens.Wiechula@ikf.uni-frankfurt.de
+///
+
+/// @brief  Simple templated 3D point
```

Comment:
I think we should not code these classes our selfs as they are part of linear algebra packages. I am in favour of defining these types globally for use in Alice-O2 and I made a proposition in PR #276. 

So possibly we can move drop these classes at some point in the future.